lib/src/task_impl.dart

```diff
@@ -1,9 +1,9 @@
 import 'package:meilisearch/src/update_status.dart';
 
 import 'index_impl.dart';
-import 'pending_update.dart';
+import 'task.dart';
 
-class PendingUpdateImpl implements PendingUpdate {
+class PendingUpdateImpl implements Task {
```

Comment:
Should `PendingUpdateImpl` be renamed too?

Reply:
Oh yeah, you're correct :D